k8s/demo/common/ia/case-api.yaml

```diff
@@ -20,11 +20,11 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/ia/case-api:prod-d0fdb179
+      image: hmctspublic.azurecr.io/ia/case-api:pr-771-818d3998
```

Comment:
do you have changes in case-api?

Reply:
if you want to use master branch code only you can leave it `hmctspublic.azurecr.io/ia/case-api:prod-d0fdb179`

Reply:
Yes there is a corresponding case-api change for this, so I have updated with the relevant case-api PR